app/views/clients/show.erb

```diff
@@ -0,0 +1,67 @@
+<div class="parallax-container" data-parallax="scroll" data-position="top" data-bleed="10" data-image-src="https://hd.unsplash.com/photo-1457090673666-0e205962d75c" data-natural-width="1400" data-natural-height="900" style="height: 300px;"></div>
+
+<div id='gitmaster' class="jumbotron" align="center">
+  <div class="container">
+    <h1 >Hey <%= @client.identifier.capitalize %>!</h1>
+    <h2>Here are your stats:</h2>
+  </div>
+</div>
+
+<div class="parallax-container" data-parallax="scroll" data-position="top" data-bleed="10" data-image-src="https://hd.unsplash.com/photo-1451756282453-567d7f57ea81" data-natural-width="1400" data-natural-height="900" style="height: 200px;"></div>
+
+<div class="jumbotron" align="center">
+  <div class="avg">
+    <h3>Average Response Time</h3>
+    <p><%= @client.payloads.average_response  %></p></br>
+  </div>
+
+  <div class="max">
+    <h3>Max Response Time</h3>
+    <p><%= @client.payloads.max_response %></p></br>
+  </div>
+
+  <div class="min">
+    <h3>Min Response Time</h3>
+    <p><%= @client.payloads.min_response %></p></br>
+  </div>
+
+  <div class="type_freq">
+    <h3>Most Frequent Response Type</h3>
+    <p><%= @client.request_types.most_frequent %></p></br>
+  </div>
+
+  <div class="verbs">
+    <h3>List of HTTP Verbs</h3>
+    <p> <%= @client.request_types.all_verbs %></p></br>
+  </div>
+
+  <div class="url_freq">
+    <h3>List of Urls</h3>
+    <ul class="list-unstyled">
+      <% @client.urls.most_to_least_requested.each do |url| %>
+      <li>
+        <a href="/sources/<%=@client.identifier%>/urls<%= URI(url).path %>">
```

Comment:
accessing a class (URI) in your views is a code smell. this is not the responsibility of the view.